src/vm/functions/assets.rs

```diff
@@ -0,0 +1,211 @@
+use vm::functions::tuples;
+use vm::functions::tuples::TupleDefinitionType::{Implicit, Explicit};
+
+use vm::types::{Value, OptionalData, BuffData, PrincipalData, BlockInfoProperty, AtomTypeIdentifier};
+use vm::representations::{SymbolicExpression};
+use vm::errors::{Error, UncheckedError, InterpreterError, RuntimeErrorType, InterpreterResult as Result, check_argument_count};
+use vm::{eval, LocalContext, Environment};
+
+const E_MINT_ASSET_ALREADY_EXIST:       i128 = 1;
```

Comment:
`enum`?

Reply:
Addressed in 0e4f022187b05dc9702641a27e4bb867d0b02fa2